site/src/components/Hero.js

```diff
@@ -0,0 +1,43 @@
+import React from "react";
+
+// assets import
+import cover from "../assets/images/hero/cover.svg";
+import icon1 from "../assets/images/hero/lee_calcote.png";
+import icon2 from "../assets/images/hero/nic_jackson.png";
+import graphic from "../assets/images/hero/graphic.svg";
+
+
+// styles import
+import HeroStyleWrapper from "../sections/hero.style";
+
+const Author = ({ src, name }) => {
+  return (
+    <div className="author">
+      <img src={src} alt={name} />
```

Comment:
Use the `StaticImage` component of gatsby for PNG images.
Reference: [Static Image](https://www.gatsbyjs.com/docs/reference/built-in-components/gatsby-plugin-image/#staticimage)

Reply:
Hi @Nikhil-Ladha,
Thank you for the feedback! I have resolved this.